pkg/domain/infra/abi/play.go

```diff
@@ -341,7 +341,7 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		}
 		named, err := reference.ParseNormalizedNamed(container.Image)
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrapf(err, "Failed to parse image \"%s\"", container.Image)
```

Comment:
Using `%q` would be more idiomatic:
```suggestion
			return nil, errors.Wrapf(err, "Failed to parse image %q", container.Image)
```

Reply:
Nice, I like that better. I updated my patch to use that.